[0m16:11:22.796643 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1043d57c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105c30550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105c30370>]}


============================== 16:11:22.799061 | 731bf56f-1039-4124-b715-db0c6927a5ae ==============================
[0m16:11:22.799061 [info ] [MainThread]: Running with dbt=1.9.4
[0m16:11:22.799391 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/feepieper/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt init ecommerce', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:11:22.806827 [debug] [MainThread]: Starter project path: /Users/feepieper/Library/Python/3.9/lib/python/site-packages/dbt/include/starter_project
[0m16:11:22.810036 [info ] [MainThread]: 
Your new dbt project "ecommerce" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:11:22.810231 [info ] [MainThread]: Setting up your profile.
[0m16:13:04.873707 [info ] [MainThread]: Profile ecommerce written to /Users/feepieper/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:13:04.877945 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 102.11986, "process_in_blocks": "0", "process_kernel_time": 0.11627, "process_mem_max_rss": "106512384", "process_out_blocks": "0", "process_user_time": 0.778535}
[0m16:13:04.878760 [debug] [MainThread]: Command `dbt init` succeeded at 16:13:04.878602 after 102.12 seconds
[0m16:13:04.879668 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1043d57c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105c1f550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105bda8e0>]}
[0m16:13:04.880389 [debug] [MainThread]: Flushing usage events
[0m16:13:06.075943 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m09:19:39.110293 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1036f6790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104549be0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104549dc0>]}


============================== 09:19:39.112919 | 8719279c-6392-4ea2-bc32-6aac48518de4 ==============================
[0m09:19:39.112919 [info ] [MainThread]: Running with dbt=1.9.4
[0m09:19:39.113227 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/feepieper/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:19:39.120527 [info ] [MainThread]: dbt version: 1.9.4
[0m09:19:39.120722 [info ] [MainThread]: python version: 3.9.6
[0m09:19:39.120879 [info ] [MainThread]: python path: /Library/Developer/CommandLineTools/usr/bin/python3
[0m09:19:39.121036 [info ] [MainThread]: os info: macOS-15.3.2-arm64-arm-64bit
[0m09:19:39.256051 [info ] [MainThread]: Using profiles dir at /Users/feepieper/.dbt
[0m09:19:39.256298 [info ] [MainThread]: Using profiles.yml file at /Users/feepieper/.dbt/profiles.yml
[0m09:19:39.256471 [info ] [MainThread]: Using dbt_project.yml file at /Users/feepieper/Documents/Ironhack/project_AWS/dbt_project.yml
[0m09:19:39.257073 [info ] [MainThread]: adapter type: snowflake
[0m09:19:39.257345 [info ] [MainThread]: adapter version: 1.9.2
[0m09:19:39.257533 [info ] [MainThread]: Configuration:
[0m09:19:39.257683 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m09:19:39.257830 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m09:19:39.257973 [info ] [MainThread]: Required dependencies:
[0m09:19:39.258156 [debug] [MainThread]: Executing "git --help"
[0m09:19:39.273870 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m09:19:39.274300 [debug] [MainThread]: STDERR: "b''"
[0m09:19:39.274488 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m09:19:39.274662 [info ] [MainThread]: Connection:
[0m09:19:39.274847 [info ] [MainThread]:   account: vi30067.gvzwdir
[0m09:19:39.274993 [info ] [MainThread]:   user: feepieper
[0m09:19:39.275137 [info ] [MainThread]:   database: SNOWPIPE_DB
[0m09:19:39.275275 [info ] [MainThread]:   warehouse: compute_wh
[0m09:19:39.275423 [info ] [MainThread]:   role: accountadmin
[0m09:19:39.275560 [info ] [MainThread]:   schema: OLTP
[0m09:19:39.275702 [info ] [MainThread]:   authenticator: None
[0m09:19:39.275841 [info ] [MainThread]:   oauth_client_id: None
[0m09:19:39.275975 [info ] [MainThread]:   query_tag: None
[0m09:19:39.276112 [info ] [MainThread]:   client_session_keep_alive: False
[0m09:19:39.276247 [info ] [MainThread]:   host: None
[0m09:19:39.276381 [info ] [MainThread]:   port: None
[0m09:19:39.276517 [info ] [MainThread]:   proxy_host: None
[0m09:19:39.276654 [info ] [MainThread]:   proxy_port: None
[0m09:19:39.276789 [info ] [MainThread]:   protocol: None
[0m09:19:39.276924 [info ] [MainThread]:   connect_retries: 1
[0m09:19:39.277063 [info ] [MainThread]:   connect_timeout: None
[0m09:19:39.277198 [info ] [MainThread]:   retry_on_database_errors: False
[0m09:19:39.277333 [info ] [MainThread]:   retry_all: False
[0m09:19:39.277467 [info ] [MainThread]:   insecure_mode: False
[0m09:19:39.277601 [info ] [MainThread]:   reuse_connections: True
[0m09:19:39.277942 [info ] [MainThread]: Registered adapter: snowflake=1.9.2
[0m09:19:39.346069 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m09:19:39.369953 [debug] [MainThread]: Using snowflake connection "debug"
[0m09:19:39.370158 [debug] [MainThread]: On debug: select 1 as id
[0m09:19:39.370307 [debug] [MainThread]: Opening a new connection, currently in state init
[0m09:19:43.088381 [debug] [MainThread]: Snowflake adapter: Error running SQL: select 1 as id
[0m09:19:43.088898 [debug] [MainThread]: Snowflake adapter: Rolling back transaction.
[0m09:19:43.089262 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m09:19:43.089650 [info ] [MainThread]: [31m2 checks failed:[0m
[0m09:19:43.089943 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/feepieper/Documents/Ironhack/project_AWS/dbt_project.yml> not found

[0m09:19:43.090248 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  250001: Could not connect to Snowflake backend after 2 attempt(s).Aborting
  Verify that the hostnames and port numbers in SYSTEM$ALLOWLIST are added to your firewall's allowed list.
  To further troubleshoot your connection you may reference the following article: https://docs.snowflake.com/en/user-guide/client-connectivity-troubleshooting/overview.

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m09:19:43.092315 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 4.0197988, "process_in_blocks": "0", "process_kernel_time": 0.166869, "process_mem_max_rss": "127811584", "process_out_blocks": "0", "process_user_time": 1.122322}
[0m09:19:43.092751 [debug] [MainThread]: Command `dbt debug` failed at 09:19:43.092655 after 4.02 seconds
[0m09:19:43.093085 [debug] [MainThread]: Connection 'debug' was left open.
[0m09:19:43.093371 [debug] [MainThread]: On debug: No close available on handle
[0m09:19:43.093692 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1036f6790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e7fd60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e7fe80>]}
[0m09:19:43.094104 [debug] [MainThread]: Flushing usage events
[0m09:19:43.555073 [debug] [MainThread]: An error was encountered while trying to flush usage events
